lib/unsanitized.js

```diff
@@ -0,0 +1,83 @@
+// names of escaping functions that we acknowledge
+const VALID_ESCAPERS = ["Sanitizer.escapeHTML", "escapeHTML"];
+const VALID_UNWRAPPERS = ["Sanitizer.unwrapSafeHTML", "unwrapSafeHTML"];
+
+class Unsanitized {
```

Comment:
The class is for holding the "context" state, its either that or rethink both functions params... as they have too many already. Both need a context which allows us to also do config parsing in this class too.

> Also, what do we get from creating a const that has the...

Er assuming you are on about the import, I import with a const just because it's something we don't want to modify in the individual rules? Can change to let if you prefer?